wiki/osu!_wiki_Contribution_Guide/GitHub_Desktop/es.md

```diff
@@ -0,0 +1,90 @@
+
+   ![](img/github-desktop-branch.jpg)
+
+3. Continua en [Editando](#editando).
+
+## Editando
+
+1. En el archivo en el que has clonado el repositorio, navega al archivo que quieres editar. Los artículos están organizados por el nombre de las carpetas siendo el nombre de los artículos y los archivos `.md` siendo las traducciones.
+2. Una vez que has encontrado el archivo correcto. ábrelo usando tu editor de texto favorito.
+3. Ahora puedes hacer tus cambios. Cuando haces cambios, sigue el [Criterio de Estilo de Artículos](/wiki/Article_Styling_Criteria) tanto como puedas.
+4. Continua en [Haciendo 'commits' y 'pushes'](#haciendo-'commits'-y-'pushes').
+
+### Moviendo archivos existentes
+
+1. Usa el Explorador de Archivos de Windows para mover archivos a cualquier parte.
+2. Continua en [Haciendo 'commits' y 'pushes'](#haciendo-'commits'-y-'pushes').
+
+### Creando nuevos archivos
+
+*Precaución: Si vas a crear artículos usando el Explorador de Archivos de Windows, asegúrate de que la opción `Extensiones de nombre de archivo` esté activa.* Vea [Cómo mostrar o esconder las extensiones de los archivos en el explorador de Windows](https://support.microsoft.com/en-us/help/865219/how-to-show-or-hide-file-name-extensions-in-windows-explorer) para las instrucciones.
+
+1. Usa el Explorador de Archivos de Windows, tu editor de imágenes favorito y/o tu editor de texto favorito para crear nuevos archivos. Si estás creando nuevos artículos o traducciones, renombra el archivo siguiendo los [Archivos en el Criterio de Estilo de Artículos](/wiki/Article_Styling_Criteria#locales).<!-- TODO: change this wikilink header according to translation in the Article_Styling_Criteria article when its created -->
+2. Continua en [Haciendo 'commits' y 'pushes'](#haciendo-'commits'-y-'pushes').
+
+### Eliminando archivos
+
+1. Usa el Explorador de Archivos de Windows para eliminar archivos.
+2. Continua en [Haciendo 'commits' y 'pushes'](#haciendo-'commits'-y-'pushes').
+
+## Haciendo 'commits' y 'pushes'
+
+1. Vuelve a GitHub para Escritorio. Debería de verse algo similar a esto:
+
+   ![](img/github-desktop-changes.jpg)
+
+2. Por defecto, GitHub para Escritorio automáticamente seleccionará todos tus cambios.
+3. Lo mejor es separar cada acción "grande" como un 'commit' aparte.
+
+Por ejemplo, como se muestra arriba, [pippi](/wiki/Mascots#pippi) ha agregado una imagen, ha modificado el archivo `en.md` para agregar un título y un link a la imagen y ha marcado las traducciones como desactualizadas. De esto podemos hacer dos 'commits', una para los cambios que afectan al archivo `en.md` y otra para marcar las traducciones como desactualizadas.
+
+4. Selecciona los archivos que vayas a añadir al 'commit'.
+5. Escribe un resumen para el commit. Ésto tiene un límite de 72 caracteres y **debe de estar en Inlés**.
+6. Escribe una descripción. Ésto es opcional, no tiene un límite de caracteres y puede ser en cualquier idioma.
```

Comment:
According to `en.md` the lines 71:72 and 76:78 are together but the check throws an error. Should I separate them or leave them like that?

Reply:
what it really meant was that if you have a list which is broken apart by an image, you need to add an empty line between all its items -- see my commit

Reply:
Thank you! I did it exactly as in `en.md` though